source/components/definition-list/definition-list.less

```diff
@@ -0,0 +1,48 @@
+@import (reference) 'grid/grid';
+hx-dl {
+  display: block;
+  font-size: 1rem; // 16px
+
+  hx-def {
+    display: flex;
+
+    hx-dt {
+      #grid.span(2);
+      color: @gray-750;
+      display: block;
+      flex-shrink: 1;
+      font-weight: 400;
+      text-align: right;
+    }
+
+    hx-dd {
+      #grid.span(10);
+      color: @gray-900;
+      display: block;
+      flex-shrink: 1;
+      margin-left: 1.25rem; // 20px
+      word-break: break-all;
+    }
+
+    + hx-def {
+      margin-top: 1.25rem; // 20px
+    }
+  }
+
+  &.hxVertical {
+    hx-def {
+      flex-direction: column;
+    }
+
+    hx-dt,
+    hx-dd {
+      #grid.span(12);
+      margin: 0;
+      text-align: left;
```

Comment:
It's best to use `inherit` instead of `left` to return it to default alignment. This makes it easier to support RTL mode in the future.

```
text-align: inherit;
```